pkg/util/limit/limiter_test.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2018 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package limit
+
+import (
+	"context"
+	"runtime"
+	"sync/atomic"
+	"testing"
+
+	"golang.org/x/sync/errgroup"
+
+	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
+)
+
+func TestConcurrentRequestLimiter(t *testing.T) {
```

Comment:
Do we need to run this test on every CI build? Seems excessive. Maybe skip it if t.Short is on?

Reply:
I thought about that, or lowering runs if t.Short was on, but it currently runs in .2s so it didn't seem that bad?